src/style.css

```diff
@@ -0,0 +1,92 @@
+@font-face {
+  font-family: Avenir;
+  font-style: normal;
+  font-weight: normal;
+  src: url(fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: Avenir;
+  font-style: normal;
+  font-weight: bold;
+  src: url(fonts/Avenir-Heavy.ttf);
+}
+
+body {
+  font-family: Avenir, sans-serif;
+  font-weight: 300;
+}
+
+.searchbar {
+  position: relative;
+  margin: 20px 0;
+}
+
+.input-searchbar {
+  box-sizing: border-box;
+  font-family: inherit;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  color: #3d4e61;
+  width: 100%;
+  border-radius: 4px;
+}
+
+.input-big-searchbar {
+  padding-left: 62px;
+  height: 70px;
+  font-size: 16px;
+  line-height: 21.86px;
+  background-image: url("images/Search.svg");
+  background-repeat: no-repeat;
+  background-size: 19px 19px;
+  background-position: 26px 25px;
+}
+
+.input-big-searchbar:hover {
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 20%);
+  background-image: url("images/Search.svg");
+  background-repeat: no-repeat;
+  background-size: 19px 19px;
+  background-position: 26px 25px;
+}
+
+.input-big-searchbar:focus {
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  border-radius: 3px;
+  font-weight: 900;
+  background-image: url("images/Search.svg");
+  background-repeat: no-repeat;
+  background-size: 19px 19px;
+  background-position: 26px 25px;
+}
+
+.input-small-searchbar {
+  font-family: inherit;
+  height: 42px;
+  padding-left: 33px;
+  font-size: 14px;
+  line-height: 19px;
+  background-image: url("images/Search.svg");
+  background-repeat: no-repeat;
+  background-size: 11px;
+  background-position: 12px 14px;
+}
+
+.input-small-searchbar:hover {
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 20%);
+  background-image: url("images/Search.svg");
+  background-repeat: no-repeat;
+  background-size: 11px;
+  background-position: 12px 14px;
+}
+
+.input-small-searchbar:focus {
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  border-radius: 3px;
+  font-weight: 900;
+  background-image: url("images/Search.svg");
+  background-repeat: no-repeat;
+  background-size: 11px;
+  background-position: 12px 14px;
+}
```

Comment:
Don't repeat yourself. The position is the same for both images. You can create a common class for it